library/swift/src/EngineBuilder.swift

```diff
@@ -1,6 +1,14 @@
 @_implementationOnly import EnvoyEngine
 import Foundation
 
+extension EnvoyNativeFilterConfig {
```

Comment:
Why not put this in objective-c since that's where the type is defined and these are primitives? If you want to keep it here, it should be `private extension`